ansible/testbed-cli.sh

```diff
@@ -41,6 +41,9 @@ function usage
   echo "To configure a VM on a server: $0 config-vm 'topo-name' 'vm-name' ~/.password"
   echo "To generate minigraph for DUT in a topology: $0 gen-mg 'topo-name' 'inventory' ~/.password"
   echo "To deploy minigraph to DUT in a topology: $0 deploy-mg 'topo-name' 'inventory' ~/.password"
+  echo "    gen-mg, deploy-mg, test-mg supports enabling data ACL with parameter"
+  echo "        -e enable_data_plane_acl=true"
+  echo "        by default, data acl is not enabled"
```

Comment:
I suggest not to change the default behavior.

Reply:
Thanks for catching this. acl test does depend on having data plane ACL enabled.

Changed the default behavior to enable data plane ACL.